src/libraries/Common/src/Interop/Windows/BCrypt/Interop.Blobs.cs

```diff
@@ -227,10 +227,27 @@ internal struct BCRYPT_ECCFULLKEY_BLOB
         }
 
         /// <summary>
-        ///     NCrypt buffer descriptors
+        ///     NCrypt or BCrypt buffer descriptors
         /// </summary>
-        internal enum NCryptBufferDescriptors : int
+        internal enum NCryptBCryptBufferDescriptors : int
```

Comment:
Consider `CngBufferDescriptors`, since "CNG" and "NCrypt or BCrypt" are... close enough to synonyms.